load.php

```diff
@@ -327,6 +328,11 @@ function perflab_load_active_and_valid_modules() {
 function perflab_maybe_set_object_cache_dropin() {
 	global $wp_filesystem;
 
+	// Bail if Server-Timing is disabled entirely.
+	if ( defined( 'PERFLAB_DISABLE_SERVER_TIMING' ) && PERFLAB_DISABLE_SERVER_TIMING ) {
+		return;
+	}
+
```

Comment:
Why does Server-Timing being disabled also disable the object cache drop-in?

Reply:
The object cache drop-in measures server timing?

Reply:
I see:

https://github.com/WordPress/performance/blob/b09405b885d485248cdb38a59f28e2007aed15ce/server-timing/object-cache.copy.php#L14-L16

That being the case, couldn't `Server-Timing` just be disabled with the existing `PERFLAB_DISABLE_OBJECT_CACHE_DROPIN` constant?

Reply:
Server-Timing can be measured with or without the drop-in though. With the drop-in we can just measure more things.
In my project I actually rely on being able to measure it without the drop-in.